packages/suite/src/actions/wallet/__tests__/coinjoinClientActions.test.ts

```diff
@@ -0,0 +1,96 @@
+import { combineReducers, createReducer } from '@reduxjs/toolkit';
+import { configureMockStore } from '@suite-common/test-utils';
+
+import { accountsReducer } from '@wallet-reducers';
+import { coinjoinReducer } from '@wallet-reducers/coinjoinReducer';
+import { onCoinjoinRoundChanged } from '../coinjoinClientActions';
+import * as fixtures from '../__fixtures__/coinjoinClientActions';
+
+jest.mock('@trezor/connect', () => global.JestMocks.getTrezorConnect({}));
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const TrezorConnect = require('@trezor/connect').default;
+
+const DEVICE = global.JestMocks.getSuiteDevice({ state: 'device-state', connected: true });
```

Comment:
Better import `testMocks` from `@suite-common/test-utils` because global mocks are kind of problematic because you need to setup it jest config and also take care TS setup for global variables etc. It's pain in the ass when moving things around packages in monorepo. With direct import you don't need to care about anything.

Reply:
thanks, changed and force pushed last commit